### DISCUSSION TITLE
Compare linearmouse with mos

### DISCUSSION THREAD

**Discussion by zhibisora** [linearmouse/linearmouse#498] [General]
What is the difference between them (mos and linearmouse)?

**Comment by lujjjh** [comment]
I'd say that the primary design goals of Mos and LinearMouse are different. You can compare the features between [Mos](https://github.com/Caldis/Mos/blob/master/README.enUS.md) and [LinearMouse](https://linearmouse.app/), or alternatively, you can try each of them separately.

**Comment by GengZhonghua** [comment]
MOS scrolls smoother like Mouse Fix.
LinearMouse adjusts the speed of mouse movement.
It would be nice if LinearMouse could scroll smoothly.
